app/controllers/health_cards_controller.rb

```diff
@@ -33,7 +43,6 @@ def qr_contents
     @jws_payload = HealthCards::Importer.scan(params[:qr_contents])
     @patient = helpers.create_patient_from_jws(@jws_payload)
   end
-
```

Comment:
Rubocop is complaining about this missing empty line:

```
app/controllers/health_cards_controller.rb:46:3: C: [Correctable] Layout/EmptyLineBetweenDefs: Expected 1 empty line between method definitions; found 0.
  def upload
  ^^^^^^^^^^

```

Reply:
resolved